GuessingGame.html

```diff
@@ -51,6 +52,105 @@
     } while (total < 3);
     if (total > 2) {
       alert("Congrats! " + name + "! You kicked ass!");
+=======
+<html>
+  <body>
+    <head>
+  <title>Bad Ass Ladies of the Supreme Court</title> </head>
+  <h1>Bad Ass <strong>Ladies</strong> of The United States <strong>Supreme Court</strong></font></h1>
+
+<form>How many Female Supreme Court Justices can you name?<br>
+  <input id="userInput" type="text" name="firstname"><br>
+<button>Submit Answer</button>
+<hr />
+
+<h2 id="sass"></h2>
+
```

Comment:
You can get rid of all these line breaks in your HTML. You want the HTML very consolidated and clean when you're done with it.